src/multiregex/__init__.py

```diff
@@ -253,18 +250,17 @@ def format_prematcher_false_positives(self, worst_n: Optional[int] = None) -> st
 def validate_prematcher(prematcher: str) -> None:
     if not prematcher or any(map(str.isupper, prematcher)):
         raise ValueError(
-            "Prematcher {!r} must be non-empty, all-lowercase, all-ASCII".format(
-                prematcher
-            )
+            f"Prematcher {prematcher!r} must be non-empty, all-lowercase, all-ASCII"
```

Comment:
is this python 3.6 compatible?

Reply:
I don't think we should actively support python versions which reached EOL